tensorrt_llm/_torch/disaggregation/native/perf_logger.py

```
@@ -315,6 +331,73 @@ def log_recv_task_perf(
+                        file_handler = logging.FileHandler(log_file, mode="a", encoding="utf-8")
+                        formatter = logging.Formatter(
+                            fmt="%(asctime)s.%(msecs)03d,%(message)s",
+                            datefmt="%Y-%m-%d %H:%M:%S",
+                        )
+                        file_handler.setFormatter(formatter)
+                        file_logger.addHandler(file_handler)
+                        if write_header:
+                            file_handler.stream.write(_GEN_SUMMARY_HEADER + "\n")
+                            file_handler.stream.flush()
+                        self._file_loggers[key] = file_logger
+                    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)